config/settings.yml

```diff
@@ -56,6 +56,13 @@ student_csv_order:
   - id_number
   - email
 
+end_user_csv_order:
```

Comment:
This is good. Any time the settings are updated, please also make sure to update the settings validations found in `config/initializers/config.rb`.